pyramid_zipkin/zipkin.py

```diff
@@ -199,3 +199,38 @@ def __exit__(self, _exc_type, _exc_value, _exc_traceback):
             binary_annotations=self.binary_annotations,
             span_id=self.span_id,
         )
+
+
+def zipkin_span(
+    service_name,
+    span_name,
+    binary_annotations=None,
+):
+    """Decorator utility for logging a function as a Zipkin span.
+    Note that the decorator doesn't accept timestamp annotations.
+    Any timestamps included in the decorator would likely be way off,
```

Comment:
Yeah... I'm fine leaving this comment out. If the intrepid decorator user is like "Why can't I add timestamped annotations?!" they can investigate further.